lib/screens/episode.dart

```diff
@@ -90,11 +90,9 @@ class _EpisodeScreenState extends ConsumerState<EpisodeScreen> with AutoRouteAwa
   @override
   void didUpdateWidget(old) {
     super.didUpdateWidget(old);
+    scrollCompleter = wrapInCompleter(scrollController.animateTo(0, duration: const Duration(milliseconds: 600), curve: Curves.easeOutExpo));
```

Comment:
you removed setState, why?

Reply:
According to the documentation, setState is not required inside didUpdateWidget.

> The framework always calls build after calling didUpdateWidget, which means any calls to setState in didUpdateWidget are redundant.

https://api.flutter.dev/flutter/widgets/State/didUpdateWidget.html